src/setup.zig

```diff
@@ -121,18 +121,18 @@ pub fn wizard(allocator: std.mem.Allocator) !void {
     defer if (local_path) |d| allocator.free(d);
     defer if (global_path) |d| allocator.free(d);
 
+    const can_access_global = blk: {
+        std.fs.accessAbsolute(global_path orelse break :blk false, .{}) catch break :blk false;
+        break :blk true;
+    };
+
     if (global_path == null and local_path == null) {
         write("Could not open a global or local config directory.\n");
         return;
     }
     var config_path: []const u8 = undefined;
-    if (try askBool("Should this configuration be system-wide?")) {
-        if (global_path) |p| {
-            config_path = p;
-        } else {
-            write("Could not find a global config directory.\n");
-            return;
-        }
+    if (can_access_global and global_path != null and try askBool("Should this configuration be system-wide?")) {
```

Comment:
As `can_access_global` is set to false above when `global_path` is null, we don't have to check if it is non-null again.